### PR TITLE
move scripts in Utilties and Validation to py3

### PR DESCRIPTION
**Pull request by davidlange6**
I do not believe that anything changed is used in unit tests...

### CODE REVIEW

**Comment by davidlange6** [comment]
as @smuzaffar suggested I ran python3 -m compileall on these and fixed the corresponding problems.

File: Utilities/ReleaseScripts/scripts/cernsso [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.6
+#!/usr/bin/env python32.6
```
Comment (makortel) [inline Utilities/ReleaseScripts/scripts/cernsso:1]: Should this be just
```suggestion
#!/usr/bin/env python3
```
? (although the current forcing to 2.6 looks suspicious)

**Comment by davidlange6** [comment]
please test

**Comment by makortel** [comment]
+1

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**